css/style.css

```diff
@@ -0,0 +1,2065 @@
+
+@font-face {
+    font-family: 'Montserrat';
+    font-style: normal;
+    font-weight: 400;
+    src: local('Montserrat Regular'), local('Montserrat-Regular'),
+         url('../fonts/montserrat-v14-latin_cyrillic-regular.woff2') format('woff2'), 
+         url('../fonts/montserrat-v14-latin_cyrillic-regular.woff') format('woff'); 
+  }
+
+  @font-face {
+    font-family: 'Montserrat';
+    font-style: normal;
+    font-weight: 500;
+    src: local('Montserrat Medium'), local('Montserrat-Medium'),
+         url('../fonts/montserrat-v14-latin_cyrillic-500.woff2') format('woff2'),
+         url('../fonts/montserrat-v14-latin_cyrillic-500.woff') format('woff'); 
+  }
+
+  @font-face {
+    font-family: 'Montserrat';
+    font-style: normal;
+    font-weight: 600;
+    src: local('Montserrat SemiBold'), local('Montserrat-SemiBold'),
+         url('../fonts/montserrat-v14-latin_cyrillic-600.woff2') format('woff2'),
+         url('../fonts/montserrat-v14-latin_cyrillic-600.woff') format('woff');
+  }
+
+  @font-face {
+    font-family: 'Montserrat';
+    font-style: normal;
+    font-weight: 700;
+    src: local('Montserrat Bold'), local('Montserrat-Bold'),
+         url('../fonts/montserrat-v14-latin_cyrillic-700.woff2') format('woff2'),
+         url('../fonts/montserrat-v14-latin_cyrillic-700.woff') format('woff'); 
+  }
+
+
+html {
+    box-sizing: border-box;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)